Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -130,6 +136,10 @@ try {
             [IO.File]::AppendAllLines($tmpFileName, [string[]]@($fullFilePath))
         }
 
+        [string] $encodedRequestName = [System.Web.HttpUtility]::UrlEncode($RequestName)
+        [string] $requestUrl = "$SymbolServiceUri/DefaultCollection/_apis/Symbol/requests?requestName=$encodedRequestName"
```

Comment:
why is the collection name hardcoded? isn't this incorrect for on prem customers when their collection name is different (or have multiple collections)?